cmd/submit_test.go

```diff
@@ -561,7 +561,7 @@ func fakeSubmitServer(t *testing.T, submittedFiles map[string]string) *httptest.
 			if err != nil {
 				t.Fatal(err)
 			}
-			submittedFiles[fileHeader.Filename] = string(body)
+			submittedFiles[filepath.Base(fileHeader.Filename)] = string(body)
```

Comment:
> So this particular change can be dropped once the Go version for running the test on CI is bumped from 1.15 to 1.17+, as fileheader.Filename will already have the directory information stripped from it.

Thanks :) Shall we do this as part of this PR?

Reply:
Given this is a small change and ready go. I would suggest merging as is and creating a new PR for bumping Go versions. 

That said, since the go.mod is still at 1.15 we may not want to remove this if the expectation is to have Go 1.15 as the target toolchain version. 

——
My rationale for a separate PR is so that we can look into adding a matrix to test against different Go versions during CI.  This would be good to validate that new changes don’t break compatibly with older supported go versions. For example the CLI releases are made with 1.19 but the CI is 1.15. A matrix can help validate each version. 

As alternative maybe @ErikSchierboom and @kytrinyx would be up for moving to a higher version of Go for dev and bumping the go.mod file version. Which also involves updating the contributing guide to call out the min dev version.

These are straight forward changes that I won’t introduce any breaking changes but will have a little meat to them. So a separate PR feel right.

Reply:
I agree that it makes sense to do it in a separate PR (and looking into a matrix seems sensible).

I don't have any objections to bumping the Go version (but also I don't actually do any real work on this anymore, so I don't have skin in the game).

Reply:
@iHiD just a heads up I'm working on a PR to bump Go versions. I will work on getting it opened this week. It currently sits as a draft at https://github.com/nywilken/exercism-cli/pull/1

Would you prefer I open a discussion on the Exercism forum before opening a PR?